tests/data/packageVersion.php

```diff
@@ -11,7 +11,6 @@
 // TODO: not-installed/package:<5 this should error because package is not in composer.json
 
 // TODO: phpunit/phpunit:<9
-// TODO: phpunit/phpunit:<10
```

Comment:
just realized after merge: next time don't delete such line but make it empty. that way you don't need unrelated changes in test expectations